code/datums/statistics/entities/survivor_survival.dm

```diff
@@ -15,12 +15,13 @@
 	round_id = GLOB.round_id || -1
 	time_after_roundstart = floor((world.time - SSticker.mode.round_time_lobby) / 600)
 	total_survivors = /datum/job/civilian/survivor::total_spawned
-	for(var/mob/living/carbon/human/human as anything in GLOB.alive_human_list)
-		if(QDELETED(human))
+	for(var/datum/weakref/ref as anything in GLOB.spawned_survivors)
+		var/mob/living/carbon/human/human = ref.resolve()
+		if(human?.stat == DEAD)
 			continue
 
-		if(issurvivorjob(human.job))
-			remaining_survivors++
+		remaining_survivors++
```

Comment:
on the offchance that a surv is gibbed, wouldn't this still increment the remaining_survivors, as human?.stat == DEAD will just resolve to false with the conditional access

Reply:
True enough. Fixing.